docker.Makefile

```diff
@@ -3,8 +3,8 @@ include common.mk
 BUILD_BASE_IMAGE = golang:1.12.6-alpine3.10
 TEST_BASE_IMAGE = golang:1.12.6
 RUN_BASE_IMAGE = alpine:3.10.0
-KUBERNETES_VERSION ?= 1.13.3
-KIND_TAG = v$(KUBERNETES_VERSION)-$(TAG)
+KUBERNETES_VERSION ?= 1.13.7
```

Comment:
Should the default be changed to Kube 1.14.x? (I see the CLI is already on 1.14; https://github.com/docker/cli/blob/83aa71c45c0f996319bfe83d697cc0514c767d90/vendor.conf#L94-L100)

Reply:
That's indeed a good point. WDYT @simonferquel @silvin-lubecki

Reply:
done